c_cxx/QNN_EP/README.md

```diff
@@ -0,0 +1,5 @@
+## How to run the application
```

Comment:
Should provide more instruction such as where to get the test model, where to place the test model, and also the qnn libs?
Should we enable the HTP test with QDQ model generation step?

Reply:
this example is strictly for cpu backend as a quick example of how to integrate with ORT. the readme gives the commandline for running the batchfile. 
the batchfile downloads the squeezenet model from onnx model zoo, builds the sample app and runs it.

Reply:
htp/qdq can be a different sample (once we've finalized our optimal quantization workflow)